openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -883,13 +891,15 @@ def process(self, instance):
         # transfer specific families from original instance to new render
         for item in self.families_transfer:
             if item in instance.data.get("families", []):
+                self.log.debug("Transfering '%s' family to instance.", item)
```

Comment:
Reading this I was kind of thinking: "transfering from where?", "why does it need transferring?", "to where?"

Should this maybe mention it's transferring it to the farm instance?
```suggestion
                self.log.debug("Transfering '%s' family to farm instance.", item)
```
Also, for the other log.

Reply:
Yeah fair, I added it for helping me figure out why some plugins weren't running remotely

Reply:
done